doc/Language/objects.pod6

```diff
@@ -835,30 +930,36 @@ class Point {
 }
 =end code
 
-However this is considered poor practice, because it makes correct
-initialization of objects from subclasses harder.
-
-Another thing to note is that the name C<new> is not special in Raku. It is
-merely a common convention, one that is followed quite thoroughly in
-L<most Raku classes|/routine/new>. You can call C<bless> from any method at
```

Comment:
I didn't remove it, but move it to  doc/Language/objects.pod6:913. I put it right before the text that explains why one shouldn't put much logic in `new`. It fits quite nicely there because the fact that `new` is only a regular method and that one shouldn't put much logic into it are related.